invite/src/invite_bot.ts

```diff
@@ -73,12 +73,34 @@ export class InviteBot {
     repo: string,
     issue_number: number,
     comment: string
-  ): Promise<void> {}
+  ): Promise<void> {
+    const macros = this.parseMacros(comment);
+
+    for (const [username, action] of Object.entries(macros)) {
+      await this.tryInvite({username, repo, issue_number, action});
+    }
+  }
 
   /**
    * Process an accepted invite by adding comments and assigning issues.
    */
-  async processAcceptedInvite(username: string): Promise<void> {}
+  async processAcceptedInvite(username: string): Promise<void> {
+    const recordedInvites = await this.record.getInvites(username);
+
+    for (const invite of recordedInvites) {
+      if (invite.action === InviteAction.INVITE) {
```

Comment:
Consider using `switch` here, with `default: throw new RangeError('Unimplemented action');` (it's more future-proof)

Reply:
I can't write a test for it (because Typescript won't let me pass a non-existent enum value), but done.